src/components/CenterContainer/CenterContainer.css

```diff
@@ -50,3 +50,63 @@
 .focus-input:focus + .focus-border {
   border: 1px solid white;
 }
+
+.focus {
+  text-align: center;
+  margin-top: 1rem;
+  font-size: 2rem;
+  font-weight: 400;
+}
+
+.focus-field {
+  margin-top: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Just a suggestion : You can make a root class for justify-align-center and use it in different places if it is occurring again and again.